bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/pageobjects/touchui/DialogConfigurer.java

```diff
@@ -72,6 +72,42 @@ public DialogField getDialogField(WebElement parentElement, String label, String
   }
 
   /**
+   * Finds the dialog field of given type within a WebElement based on the provided label. If label is not
+   * present, returns the first field from the tab.
+   *
+   * @param parentElement parent element from which DialogField will be retrieved
+   * @param label         of the requested field
+   * @param type          Class of the requested field
+   * @return DialogField of the given type based on the provided info
+   */
+  public DialogField getDialogField(WebElement parentElement, String label, Class type) {
```

Comment:
After reviewing the whole `DialogConfigurer` - do we really need 'overloaded' `getDialogField` method taking a `Class` as parameter? 
* For OOTB types in TouchUI module, you can invoke the existing method passing as argument e.g. `FieldType.MULTIFIELD.name()`
* a lot of code is duplicated here